packages/inter-protocol/test/vaultFactory/test-vaultLiquidation.js

```diff
@@ -1864,12 +1864,12 @@ test('reinstate vault', async t => {
 
   aliceUpdate = await E(aliceNotifier).getUpdateSince();
   t.is(aliceUpdate.value.vaultState, Phase.LIQUIDATED);
-  t.deepEqual(aliceUpdate.value.locked, aeth.make(0n));
+  t.deepEqual(aliceUpdate.value.locked, aeth.makeEmpty());
 
-  // Reduce Bob's collateral by liquidation penalty
+  // Reduce Bob's collateral by debt and liquidation penalty
   const recoveredBobCollateral = AmountMath.subtract(
     bobCollateralAmount,
-    aeth.make(4n),
+    aeth.make(5n),
```

Comment:
I can imagine this functionality regressing and someone reading the 1n change as a change in rounding.

Consider making the quantities in this test larger (100x or more) and making this number an expression of the collateral and the penalty rate.

Reply:
I would like to put that off for tonight. Is it worth an issue?  It seems like a reasonable cleanup.